db/migrate/20180322041822_make_transactions_act_as_polymorphic.rb

```diff
@@ -0,0 +1,8 @@
+class MakeTransactionsActAsPolymorphic < ActiveRecord::Migration[5.1]
+  def change
+    add_column :transactions, :transactinable_type, :string
+    add_column :transactions, :transactinable_id, :integer
+    remove_column :transactions, :comment, :string
+    remove_column :transactions, :date, :date
```

Comment:
@sinpai let's not just remove data - we'll lose it. Let's migrate it to a `BalanceTransactions`
see https://github.com/marderer/personal-finance/pull/35#discussion_r177036383